src/libraries/System.Drawing.Common/src/System/Drawing/ClientUtils.cs

```diff
@@ -111,7 +112,7 @@ public override bool Equals(object obj)
                 {
                     if (InnerList[i] != other.InnerList[i])
                     {
-                        if (InnerList[i] == null || !InnerList[i].Equals(other.InnerList[i]))
+                        if (InnerList[i] == null || !InnerList[i]!.Equals(other.InnerList[i]))
```

Comment:
We normally add comment to the compiler issue so that remove `!` when issue is resolved:
`// TODO-NULLABLE: https://github.com/dotnet/roslyn/issues/34644`

Reply:
Sure, although I find this unlikely to happen in the future, primarily because of the reason stated here https://github.com/dotnet/roslyn/issues/34644#issuecomment-479176405